File: test/fixtures-units/sample.py
import os


class Greeter:
    """Doc with def inside: def fake(): pass"""

    def __init__(self, name):
        self.name = name

    def greet(self, extra=None):
        def inner():
            return "{"
        if extra:
            return inner()
        return f"hi {self.name}"


def helper(a,
           b):
    return a + b


async def fetch(
    url,
):
    data = {
        'k': 1,
    }
    return data


VALUE = helper(
    1, 2)

File: test/fixtures-units/sample.swift
import Foundation

struct Point: Equatable {
    var x: Int
    var total: Int {
        get { return x }
        set { x = newValue }
    }
}

final class Greeter {
    func greet(_ name: String) -> String {
        let raw = #"has "quotes" and { brace"#
        return "hi \(name.uppercased()) {"
    }

    init(x: Int) {
        if let y = Optional(x) {
            print(y)
        }
    }
}

extension Greeter {
    func bye() { print("bye") }
}
